site/docs/education/tutorials/postman-tutorial/overview.md

```diff
@@ -85,9 +138,9 @@ Egeria platform. Tutorial content will walk you through this explicitly.
 
 * Make the egeria environment the default in this workspace
 
-![Select Egeria environment](postman-env-select.png)
-![Edit Egeria environment](postman-env-editselect.png)
-![View and edit environment values](postman-env-edit.png)
+![Select Egeria environment](postman-env-switch.png)
```

Comment:
these screen shots should be associated with text and maybe in a twisty (section)

Reply:
I've given more detailed instructions, and included the screenshot in a collapsible section.